src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -185,6 +189,19 @@ def _validate_local_requirements(stack: "Stack") -> Tuple[bool, str]:
             # this, but just in case
             assert container_registry is not None
 
+            if (
```

Comment:
After thinking about it some more, I really don't like this. This even breaks stuff like accessing secrets of the component as that currently checks whether the component is part of the active stack, and probably many more issues that we can't think of right now.

Reply:
I added a comment to explain what that new configuration attribute does. Here it is: 
```This is a temporary solution to allow the Kubeflow orchestrator
    # to be provisioned as an individual component (i.e. same as running
    # `zenml orchestrator kubeflow up`) rather than needing it to be part of the
    # active stack (i.e. instead of running `zenml stack up`). This is required
    # by the test framework because the way it works is that it first provisions
    # the stack components individually, then each test gets a different stack
    # with the components it requires.```

Reply:
We should make it clear that users should never set this themselves then